### PR TITLE
Feature player cash

### PR DESCRIPTION
**Pull request by jacobwilki01**
image

### CODE REVIEW

**Review by modmo22 — APPROVED** [review]

File: CardsCashCasino/Manager/BettingManager.cs [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+
+                IndicatorDigit indicatorDigit = new IndicatorDigit();
+                indicatorDigit.Update(digit);
+                _digits.Add(indicatorDigit);
+            }
+
+            _digits.Reverse(); // put them in the correct order
+
+            if (_topLeftCorner is not null)
+                CalculateDigitPositions(((Point)_topLeftCorner).X + 21, ((Point)_topLeftCorner).Y);
```
Comment (eBerkley) [inline CardsCashCasino/Manager/BettingManager.cs:135]: Wait I'm confused, where is the constant `21` coming from here?

Comment (jacobwilki01) [reply CardsCashCasino/Manager/BettingManager.cs:135]: its the width of a digit object. ill move it to constants.

**Review by eBerkley — APPROVED** [review]
Looks good

File: CardsCashCasino/Manager/BettingManager.cs (hunk 1 above)
Comment (jacobwilki01) [reply CardsCashCasino/Manager/BettingManager.cs:135]: going to resolve in my next ticket so this can get merged.

**Pull request merged**